makefiles/file-cmds.mk

```diff
@@ -9,13 +9,13 @@ STRAPPROJECTS     += file-cmds
 ifeq ($(shell [ "$(CFVER_WHOLE)" -lt 1600 ] && echo 1),1)
 FILE-CMDS_VERSION := 272.250.1
 else
-FILE-CMDS_VERSION := 287.40.2
+FILE-CMDS_VERSION := 352.40.6
 endif
-DEB_FILE-CMDS_V   ?= $(FILE-CMDS_VERSION)-3
+DEB_FILE-CMDS_V   ?= $(FILE-CMDS_VERSION)
```

Comment:
Move this into the `if` statement above. Would you also mind taking a look at using the `mtree` from `file-cmds` instead of from `mtree-netbsd`?